src/helpers.ts

```diff
@@ -18,7 +18,9 @@ export function pathToJest(pluginSettings: IPluginSettings) {
     const defaultCreateReactPathWindows = 'node_modules/react-scripts/node_modules/.bin/jest.cmd';
     const createReactPath = (platform() === 'win32') ? defaultCreateReactPathWindows : defaultCreateReactPath;
     const absolutePath = pluginSettings.rootPath + '/' + createReactPath;
-    if (!existsSync(path) && existsSync(absolutePath)) {
```

Comment:
I think something changed in CRA which includes jest in the `node_modules/.bin` now, or maybe I was just doing it wrong from the get-go. Now we just look for the CRA version outright and assume that must make it a CRA.